app/dashboard/poh_utils.py

```diff
@@ -1,7 +1,18 @@
+import json
+
 POH_CONTRACT_ADDRESS = '0xC5E9dDebb09Cd64DfaCab4011A0D5cEDaf7c9BDb'
 
 _poh_contract = None
 
+EIP_1271_ABI = '[{"constant":true,"inputs":[{"name":"_messageHash","type":"bytes32"},{"name":"_signature","type":"bytes"}],"name":"isValidSignature","outputs":[{"name":"magicValue","type":"bytes4"}],"payable":false,"stateMutability":"view","type":"function"}]'
```

Comment:
Super tiny, but you could change this to the same format as below https://github.com/gitcoinco/web/blob/bbac82ff2bfb8755550f59c1872befac00bfd7ff/app/dashboard/poh_utils.py#L29 and not have to use `json.loads()`

Reply:
Good point, ty!